src/benchmarks/real-world/Roslyn/StageBenchmarks.cs

```diff
@@ -18,11 +20,36 @@ public class StageBenchmarks
         private CommonPEModuleBuilder _moduleBeingBuilt;
         private EmitOptions _options;
         private MemoryStream _peStream;
+        private ParseOptions _parseOptions;
+        private SourceText[] _files;
+
+        [GlobalSetup(Target = nameof(Parsing))]
+        public void ParsingSetup()
+        {
+            var projectDir = Environment.GetEnvironmentVariable(Helpers.TestProjectEnvVarName);
+            var responseFile = Path.Combine(projectDir, "repro.rsp");
+            var cmdLineParser = new CSharpCommandLineParser();
+            var args = cmdLineParser.Parse(new[] { "@" + responseFile }, projectDir, sdkDirectory: null);
+            _parseOptions = args.ParseOptions;
+            _files = new SourceText[args.SourceFiles.Length];
+            Parallel.For(0, _files.Length, index =>
+            {
+                using (var fstream = new FileStream(args.SourceFiles[index].Path, FileMode.Open))
+                {
+                    _files[index] = SourceText.From(fstream);
+                }
+            });
+        }
 
         [Benchmark]
-        public object Parsing()
+        public SyntaxTree[] Parsing()
         {
-            return Helpers.CreateReproCompilation();
```

Comment:
Should the `CreateReproCompilation` method be removed now because it's not used anymore?